pennylane/pulse/hardware_hamiltonian.py

```diff
@@ -160,9 +158,11 @@ def circuit_local(params):
             H = \frac{1}{2} \Omega(t) \sum_{j \in \text{wires}} \left(e^{i \phi(t)} \sigma^+_j + e^{-i \phi(t)} \sigma^-_j \right)
             - \frac{1}{2} (\nu - \omega_q) \sum_{j \in \text{wires}} \sigma^z_j
 
-        We can define :math:`\Delta = \nu - \omega_q` to arrive at the definition above. Note that a potential anharmonicity term,
-        as is common for transmon systems when taking into account higher energy levels,
-        is unaffected by this transformation.
+        The latter formulation is more common in neutral atom systems where we define the detuning from the atomic energy gap
+        as :math:`\Delta = \nu - \omega_q`. This is because here all atoms have the same energy gap, whereas for superconducting
+        qubits that is typically not the case.
+        Note that a potential anharmonicity term, as is common for transmon systems when taking into account higher energy
+        levels, is unaffected by this transformation.
```

Comment:
Could this go a little bit more in depth about the new formulation? I.e. conclude with something about "this function uses the latter expression and omits the shift due to detuning, effectively setting frequency $= \omega_q$. A shift term can be added to this to account for detuning."